pymatgen/io/shengbte.py

```diff
@@ -0,0 +1,156 @@
+# coding: utf-8
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License
+import warnings
+
+from monty.dev import requires
+from monty.json import MSONable
+
+try:
+    import f90nml
+except:
+    f90nml = None
+
+"""
+This module defines IO ShengBTE for reading, updating, and writing the
+CONTROL input file
+"""
+
+__author__ = "Rees Chang"
+__copyright__ = "Copyright 2019, The Materials Project"
+__version__ = "0.1"
+__email__ = "rc564@cornell.edu"
+__date__ = "June 27, 2019"
+
+
+class Control(dict, MSONable):
+
+    """
+    Class for reading, updating, and writing ShengBTE CONTROL files.
+    """
```

Comment:
The class docstring should contain the init argument definitions. E.g. see https://google.github.io/styleguide/pyguide.html#384-classes

Basically just move the whole of the init docstring to this docstring and don't put a docstring for init.